Bootstraping a driver

Some things that I found missing (it could be maybe a lack of documentation)

### Driver
Inside the driver, after running `bblfsh-sdk prepare-build`
- I think it should be autogenerated:
  - `native` directory if it does not exists,
  - `.gitignore` with a `.sdk` rule; if `.gitignore` exists without that rule, then append it.
- It should be great a hint in the root `Makefile` when running `make` if the SDK has not been yet installing, just saying something like:
_"It is needed to instal the bblfsh SDK before; it can be done with: `bblfsh-sdk prepare-build`"_
- It should be explained somewhere that to start working with a driver, it is needed to run, in order:
```
bblfsh-sdk prepare-build
bblfsh-sdk init
```

### SDK
Inside `bblfsh/sdk` project
- `make test` does not pass
```
> make test
--- FAIL: github.com/bblfsh/sdk/etc/skeleton/driver/normalizer :: TestNativeBinary
        Error Trace:    normalizer_test.go:18
	Error:		Expected nil, but got: &os.PathError{Op:"fork/exec", Path:"/opt/driver/src/build/native", Err:0x2}
```
- is it needed to run `make all` before doing anything? anyway tests keeps failing

--- a/assets/skeleton/bindata.go
+++ b/assets/skeleton/bindata.go
@@ -881,7 +881,7 @@ func license() (*asset, error) {
 
 var _makefile = []byte(`-include .sdk/Makefile
 
-$(if $(filter true,$(sdkloaded)),,$(error You must install bblfsh-sdk))
+$(if $(filter true,$(sdkloaded)),,$(error You must install bblfsh-sdk with: "bblfsh-sdk prepare-build"))
 
 test-native-internal:
 	cd native; \
@@ -904,7 +904,7 @@ func makefile() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "Makefile", size: 328, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
+	info := bindataFileInfo{name: "Makefile", size: 361, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
 	a := &asset{bytes: bytes, info: info}
 	return a, nil
 }

--- a/assets/skeleton/bindata.go
+++ b/assets/skeleton/bindata.go
@@ -12,6 +12,7 @@
 // etc/skeleton/driver/normalizer/tonode.go
 // etc/skeleton/git/hooks/pre-commit
 // etc/skeleton/manifest.toml.tpl
+// etc/skeleton/native/README.md.tpl
 // DO NOT EDIT!
 
 package skeleton
@@ -1144,6 +1145,28 @@ func manifestTomlTpl() (*asset, error) {
 	return a, nil
 }
 
+var _nativeReadmeMdTpl = []byte(`The native driver (the one producing the native AST) should be located here.
+
+See:
+
+https://doc.bblf.sh/driver/sdk.html
+`)
+
+func nativeReadmeMdTplBytes() ([]byte, error) {
+	return _nativeReadmeMdTpl, nil
+}
+
+func nativeReadmeMdTpl() (*asset, error) {
+	bytes, err := nativeReadmeMdTplBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	info := bindataFileInfo{name: "native/README.md.tpl", size: 120, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
+	a := &asset{bytes: bytes, info: info}
+	return a, nil
+}
+
 // Asset loads and returns the asset for the given name.
 // It returns an error if the asset could not be found or
 // could not be loaded.
@@ -1208,6 +1231,7 @@ var _bindata = map[string]func() (*asset, error){
 	"driver/normalizer/tonode.go":     driverNormalizerTonodeGo,
 	"git/hooks/pre-commit":            gitHooksPreCommit,
 	"manifest.toml.tpl":               manifestTomlTpl,
+	"native/README.md.tpl":            nativeReadmeMdTpl,
 }
 
 // AssetDir returns the file names below a certain
@@ -1271,6 +1295,9 @@ var _bintree = &bintree{nil, map[string]*bintree{
 		}},
 	}},
 	"manifest.toml.tpl": &bintree{manifestTomlTpl, map[string]*bintree{}},
+	"native": &bintree{nil, map[string]*bintree{
+		"README.md.tpl": &bintree{nativeReadmeMdTpl, map[string]*bintree{}},
+	}},
 }}
 
 // RestoreAsset restores an asset under the given directory

--- a/etc/skeleton/Makefile
+++ b/etc/skeleton/Makefile
@@ -1,6 +1,6 @@
 -include .sdk/Makefile
 
-$(if $(filter true,$(sdkloaded)),,$(error You must install bblfsh-sdk))
+$(if $(filter true,$(sdkloaded)),,$(error You must install bblfsh-sdk with: "bblfsh-sdk prepare-build"))
 
 test-native-internal:
 	cd native; \

--- a/etc/skeleton/native/README.md.tpl
+++ b/etc/skeleton/native/README.md.tpl
@@ -1,0 +1,5 @@
+The native driver (the one producing the native AST) should be located here.
+
+See:
+
+https://doc.bblf.sh/driver/sdk.html

